docs/metadata.md

```diff
@@ -0,0 +1,11 @@
+# Returns metadata describing the VDS data
+
+Retrieve basic information related to the VDS data such as axis definitions, overall shape and size, etc." 
+
+## Response
+Content-Type: application/json
```

Comment:
Italic from my comment didn't copy well apparently :smile: 
This is italic on other endpoints.